### PR TITLE
bpo-41205: Document Decimal power 0 to the 0

### PR DESCRIPTION
**Pull request by srinivasreddy**
https://bugs.python.org/issue41205

### CODE REVIEW

File: Doc/library/decimal.rst [hunk 1]
```diff
@@ -1355,7 +1355,11 @@ In addition to the three supplied contexts, new contexts can be created with the
       must be integral.  The result will be inexact unless ``y`` is integral and
       the result is finite and can be expressed exactly in 'precision' digits.
       The rounding mode of the context is used. Results are always correctly-rounded
-      in the Python version.
+      in the Python version. Also note that ``Decimal(0) ** Decimal(0)`` results in
+      ``InvalidOperation``, and if ``InvalidOperation`` is not trapped, then results in
+      ``Decimal('NaN')``.
+
```
Comment (mdickinson) [inline Doc/library/decimal.rst]: Nitpick: please remove the extra blank lines here.

File: Doc/library/decimal.rst [hunk 2]
```diff
@@ -1355,7 +1355,11 @@ In addition to the three supplied contexts, new contexts can be created with the
       must be integral.  The result will be inexact unless ``y`` is integral and
       the result is finite and can be expressed exactly in 'precision' digits.
       The rounding mode of the context is used. Results are always correctly-rounded
-      in the Python version.
+      in the Python version. Also note that ``Decimal(0) ** Decimal(0)`` results in
```
Comment (mdickinson) [inline Doc/library/decimal.rst]: I'd suggest dropping the "Also note that" words; I don't think they add value.
```suggestion
      in the Python version. ``Decimal(0) ** Decimal(0)`` results in
```

**Review by mdickinson — CHANGES_REQUESTED** [review]
LGTM. One nitpicky request to remove the extra blank lines introduced in this PR, and one suggested minor wording change.

**Comment by srinivasreddy** [comment]
@mdickinson   ✅ . Thanks.

**Review by mdickinson — APPROVED** [review]
Thanks for the updates! LGTM. I'll leave this open for a bit in case @skrah or @rhettinger wants to comment.

**Review by skrah — APPROVED** [review]

**Comment by skrah** [comment]
Looks good, thanks!

File: Doc/library/decimal.rst [hunk 3]
```diff
@@ -1355,7 +1355,9 @@ In addition to the three supplied contexts, new contexts can be created with the
       must be integral.  The result will be inexact unless ``y`` is integral and
       the result is finite and can be expressed exactly in 'precision' digits.
       The rounding mode of the context is used. Results are always correctly-rounded
-      in the Python version.
+      in the Python version. ``Decimal(0) ** Decimal(0)`` results in
```
Comment (rhettinger) [inline Doc/library/decimal.rst]: Please add a blank line so this gets its own paragraph.

**Review by rhettinger — CHANGES_REQUESTED** [review]

**Comment by srinivasreddy** [comment]
@rhettinger  Done Raymond. Thanks!!!

**Comment by skrah** [comment]
@rhettinger's comment has been addressed, so I think we can merge.

**Pull request merged**